### ISSUE TITLE
Not working on Node v0.11.16

### ISSUE THREAD

**Issue by kjda** [mash/node-imagemagick-native#76]
This module works fine on node v0.10.* but if I try to use it with v0.11.16, I get the following error when I run the application. & yes, I did reinstall all node modules after switching to v0.11.16

```
module.js:519
    paths.unshift(path.resolve(homeDir, '.node_modules'));
Error: Module did not self-register.
    at Error (native)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/var/www/apps/node_modules/imagemagick-native/index.js:1:80)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
```

**Comment by CREEATION** [comment]
:+1:

**Comment by elad** [comment]
Yep, seeing this too. I'm working on making this module work with node v0.12, just need to find the time to finish it up and test...

**Comment by elad** [comment]
@geta6 fixed it first in #79. :)
